### PR TITLE
old logging API moved to legacy package

### PR DESCRIPTION
**Pull request by hendrikebbers**
as preparation for https://github.com/hashgraph/hedera-services/issues/5424

### CODE REVIEW

**Review by cody-littley — APPROVED** [review]
Validator code currently living in the regression repo currently depends on log payloads, so make sure you also create a branch there. These branches should be merged simultaneously (or as near to simultaneous as possible).

**Review by netopyr — APPROVED** [review]
LGTM in regards to `hedera-node`

**Review by OlegMazurov — APPROVED** [review]

**Review by cody-littley — DISMISSED** [review]
Signing off for hashgraph team. Just be careful before you merge this that it isn't going to cause breakages in the JRS tests.

**Comment by hendrikebbers** [comment]
@cody-littley since this PR passed the JSR tests, how should it destroy them?

**Review by povolev15 — APPROVED** [review]
LGTM

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/extensions/BusyTime.java
Comment (cody-littley) [inline platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/extensions/BusyTime.java:1]: This class was renamed and refactored. Its new name is `FractionalTimer`. I'm guessing you started this branch before the rename merged, and now git wants to add the file back in again.

**Review by timo0 — DISMISSED** [review]

File: platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/extensions/BusyTime.java
Comment (hendrikebbers) [reply platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/extensions/BusyTime.java:1]: solved

**Review by lpetrovic05 — DISMISSED** [review]

**Review by cody-littley — DISMISSED** [review]

**Review by netopyr — DISMISSED** [review]
LGTM in regards to hedera-node

**Review by povolev15 — APPROVED** [review]

**Review by OlegMazurov — DISMISSED** [review]

**Review by timo0 — APPROVED** [review]

**Review by netopyr — DISMISSED** [review]
LGTM

**Review by cody-littley — APPROVED** [review]

**Review by JeffreyDallas — APPROVED** [review]

**Review by OlegMazurov — APPROVED** [review]

**Review by nathanklick — DISMISSED** [review]

**Review by timo0 — APPROVED** [review]

**Review by cody-littley — APPROVED** [review]

**Review by imalygin — APPROVED** [review]

**Review by netopyr — APPROVED** [review]

**Pull request merged**